go/vt/schemamanager/tablet_executor.go

```diff
@@ -49,9 +52,11 @@ type TabletExecutor struct {
 }
 
 // NewTabletExecutor creates a new TabletExecutor instance
-func NewTabletExecutor(requestContext string, wr *wrangler.Wrangler, waitReplicasTimeout time.Duration) *TabletExecutor {
+func NewTabletExecutor(requestContext string, ts *topo.Server, tmc tmclient.TabletManagerClient, logger logutil.Logger, waitReplicasTimeout time.Duration) *TabletExecutor {
```

Comment:
I removed the reference to `wrangler` here and replaced it with `topo.Server` and `tabletManagerClient` to break an import cycle that will be introduced in this PR. 

``` 
    grpcvtctldserver  --> schema manager --> tablet_executor --> wrangler --> grpcvtctldserver
```